site/templates/nav/user_logged_in.jet

```diff
@@ -3,10 +3,15 @@
     <s72-dropdown class="navbar-nav-user-nav">
       <div>
         <a href="#" class="s72-btn s72-dropdown-toggle">
-          {{i18n("nav_signed_in_as")}} <s72-username></s72-username>
+          <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="user-icon" width="24" height="24" x="0px" y="0px" viewBox="-243 245 24 24" xml:space="preserve">
```

Comment:
We use Font Awesome 4 for icons... no need for svg chaos!

Reply:
Replaced with Font Awesome

Reply:
@PeculiarGoat I tried both of these and they didn't work:
https://fontawesome.com/v4.7/icon/user-circle-o
https://fontawesome.com/v4.7/icon/user-circle

This one does but looks bad:
https://fontawesome.com/v4.7/icon/user

Can we stick with the svg temporarily until we get material design icons working?